CHANGELOG.md

```diff
@@ -9,6 +9,10 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Added
 
+- Currently pulls in live converstion rate from XDR to ICP, via a canister query method `xdr_permyriad_per_icp` from: `./wallet_ui/declarations/xdr`.
```

Comment:
Sorry, I wasn't clear. We got the candid declaration from another location. If we need to upgrade the interface in the future, we should leave some sort of reference to how we got it so we can go fetch the latest version

Reply:
- Currently pulls XDR to ICP conversion rate from `nns-dapp/frontend/ts/src/canisters/cyclesMinting/canister.did` that in turn queries from CMC (cycles minting canister): "rkp4c-7iaaa-aaaaa-aaaca-cai".

Reply:
Currently pulls live XDR to ICP conversion rate via declaration sourced from `nns-dapp/frontend/ts/src/canisters/cyclesMinting/canister.did`. This in turn queries the cycles minting canister ("rkp4c-7iaaa-aaaaa-aaaca-cai").